packages/renderer/src/lib/volume/VolumesList.svelte

```diff
@@ -185,6 +163,48 @@ async function fetchUsageData() {
 function gotoCreateVolume(): void {
   router.goto('/volumes/create');
 }
+
+let selectedItemsNumber: number;
+let table: Table;
+
+const columns: Column[] = [
+  new Column('Status', 'center', '70px'),
+  new Column('Name', 'left', '3fr'),
+  new Column('Age', 'left', '1fr'),
+  new Column('Size', 'right', '1fr'),
+];
+
+const tableHelper = new (class extends TableHelper {
+  selectable(volume: any): boolean {
+    return volume.inUse;
+  }
+
+  disabledText(): string {
+    return 'Volume is used by a container';
+  }
+
+  compare(column: string): ((object1: any, object2: any) => number) | undefined {
+    if (column === 'Status') {
+      return (a, b) => {
+        let au: boolean = a.inUse;
+        let bu: boolean = b.inUse;
+        return au === bu ? 0 : au ? -1 : 1;
+      };
+    } else if (column === 'Name') {
+      return (a, b) => a.shortName.localeCompare(b.shortName);
+    } else if (column === 'Age') {
+      return (a, b) => {
+        const au = moment().diff(a.created);
+        const bu = moment().diff(b.created);
+        return au - bu;
+      };
+    } else if (column === 'Size') {
+      // TODO: should sort by value, not alphabetically (e.g. 1 Gb > 2 Mb)
+      //return (a, b) => a.humanSize.localeCompare(b.humanSize);
```

Comment:
I would expect to not have any, any in signature but a typed field so I know what I can get from there

Reply:
Fixed in latest commit.